seed_services_cli/main.py

```diff
@@ -35,7 +36,7 @@ class ci(object):
 @click.group(name="seed-services-cli")
 @click.version_option()
 @click_config.wrap(module=config, sections=('hub', 'identity_store',
-                   'stage_based_messaging', 'auth', 'ci'))
+                                            'stage_based_messaging', 'auth', 'cli'))
```

Comment:
I think the `ci` to `cli` change is incorrect. I think the `ci` section is for the control interface.